src/main/java/de/muenchen/dave/services/messstelle/GanglinieService.java

```diff
@@ -0,0 +1,250 @@
+                    setRangeMaxRoundedToHundredInZaehldatenStepline(ladeZaehldatenStepline, intervall.getAnzahlRad());
+
+                    setSeriesIndexForFirstChartValue(seriesEntries.getSeriesEntryKfz());
+                    seriesEntries.getSeriesEntryKfz().getYAxisData().add(intervall.getSummeKraftfahrzeugverkehr());
+                    setLegendInZaehldatenStepline(ladeZaehldatenStepline, ChartLegendUtil.KFZ);
+                    setRangeMaxRoundedToHundredInZaehldatenStepline(ladeZaehldatenStepline, intervall.getSummeKraftfahrzeugverkehr());
+
+                    setSeriesIndexForFirstChartValue(seriesEntries.getSeriesEntrySv());
+                    seriesEntries.getSeriesEntrySv().getYAxisData().add(intervall.getSummeSchwerverkehr());
+                    setLegendInZaehldatenStepline(ladeZaehldatenStepline, ChartLegendUtil.SCHWERVERKEHR);
+                    setRangeMaxRoundedToHundredInZaehldatenStepline(ladeZaehldatenStepline, intervall.getSummeSchwerverkehr());
+
+                    setSeriesIndexForFirstChartPercent(seriesEntries.getSeriesEntrySvProzent());
+                    seriesEntries.getSeriesEntrySvProzent().getYAxisData().add(BigDecimal.valueOf(intervall.getProzentSchwerverkehr()));
+                    setLegendInZaehldatenStepline(ladeZaehldatenStepline, ChartLegendUtil.SCHWERVERKEHR_ANTEIL_PROZENT);
+                    setRangeMaxPercentInZaehldatenStepline(ladeZaehldatenStepline, BigDecimal.valueOf(intervall.getProzentSchwerverkehr()));
+
+                    setSeriesIndexForFirstChartValue(seriesEntries.getSeriesEntryGv());
+                    seriesEntries.getSeriesEntryGv().getYAxisData().add(intervall.getSummeGueterverkehr());
+                    setLegendInZaehldatenStepline(ladeZaehldatenStepline, ChartLegendUtil.GUETERVERKEHR);
+                    setRangeMaxRoundedToHundredInZaehldatenStepline(ladeZaehldatenStepline, intervall.getSummeGueterverkehr());
+
+                    setSeriesIndexForFirstChartPercent(seriesEntries.getSeriesEntryGvProzent());
+                    seriesEntries.getSeriesEntryGvProzent().getYAxisData().add(BigDecimal.valueOf(intervall.getProzentGueterverkehr()));
+                    setLegendInZaehldatenStepline(ladeZaehldatenStepline, ChartLegendUtil.GUETERVERKEHR_ANTEIL_PROZENT);
+                    setRangeMaxPercentInZaehldatenStepline(ladeZaehldatenStepline, BigDecimal.valueOf(intervall.getProzentGueterverkehr()));
+
+                    ladeZaehldatenStepline.setXAxisDataFirstChart(
+                            ZaehldatenProcessingUtil.checkAndAddToXAxisWhenNotAvailable(
+                                    ladeZaehldatenStepline.getXAxisDataFirstChart(),
+                                    intervall.getUhrzeitVon().toString()));
+                });
+        ladeZaehldatenStepline.setSeriesEntriesFirstChart(seriesEntries.getChosenStepLineSeriesEntries());
+        return ladeZaehldatenStepline;
```

Comment:
sehe hier sehr große ähnlichkeit mit `ProcessZaehldatenSteplineService.java`, daher die frage ob wir hierfür nicht kleine funktionen schreiben können die wir in beiden Files verwenden.

Reply:
Aufgrund der Budgetknappheit wird das Refactoring verschoben